topics/setup.md

```diff
@@ -23,7 +23,7 @@ Before you begin [creating your first application](create-first-app.md) to work
     
     Check out [KMM plugin release notes](kmm-plugin-releases.md).
     
-6. Install the [JDK](https://jdk.java.net/14/) if you haven't already done so.  
```

Comment:
By the old link it was said that "JDK 14 has been superseded. Please visit jdk.java.net for the current version."
So I would offer to point a user to the most fresh versions.